Add workaround for old git version for vm (#31)
kyma-project/serverless-manager: module-chart/charts/webhook/README.md

# Webhook

## Overview

This project contains the chart for the Webhook.

## Details

> Read the webhooks readme (https://github.com/kyma-project/kyma/tree/main/components/function-controller/README.md) to get all details.

Reviewer: ```suggestion
For more information, read the Webhooks [README](https://github.com/kyma-project/kyma/tree/main/components/function-controller/README.md) file.
```
